src/graphql/queries/colony.graphql

```diff
@@ -85,3 +85,9 @@ query GetColonyMemberInvite($id: ID!) {
     invitesRemaining
   }
 }
+
+subscription OnUpdateColony {
+  onUpdateColony {
+    ...Colony
+  }
+}
```

Comment:
Big thanks to @chmanie for this... 🍺

Reply:
This is actually the first subscription that's actually being used :tada: